kbs/Makefile

```diff
@@ -34,6 +34,17 @@ passport-resource-kbs:
 cli:
 	cargo build -p kbs-client --locked --release --no-default-features --features $(CLI_FEATURES)
 
+.PHONY: cli-static-x86_64-linux
+cli-static-x86_64-linux:
```

Comment:
I'm still unsure it should be hardcoded the arch as it's not cross compiling to x86_64, it is?

Another thing is whether it should specify the target `--target=x86_64-unknown-linux-gnu` as one might just static compile with musl, for example.

Would be possible to statically build with `cli` target but tweaking options via `RUSTFLAGS`?

Reply:
When this target is executed on our github runner, it's not cross compiling, as you point out.

Statically building with musl could be cool/useful/better. I didn't try this. I'm not partial to gnu libc or musl, but bear in mind musl would also need to be present. Do you want me to add support for musl, as well, or do you want to support both? (I'm OK either way.)

I thought it better to make an explicit target than to leverage the `cli` target. If someone clones the repo and tries to build kbs-client, I want it to just work. If the dependencies aren't there, it should either fail gracefully or have some meaningful way to grab the dependencies. So, `RUSTFLAGS` would be a little cleaner (because it would just re-use the `cli` target), but there'd be all sorts of unsupported combinations there. For instance, `cli` supports `$(CLI_FEATURES)`, but we're only expecting `sample_only` to work. Similarly, we don't have any other checks for the host machine or the cross compilation libraries for building for other targets. Hence, I tried to make an explicit Makefile target with explicit flags.

Reply:
Follow-up: I thought your idea might be better, so I tried it. It runs into a weird issue:
```
$ RUSTFLAGS="--target=x86_64-unknown-linux-gnu -C target-feature=+crt-static" make cli CLI_FEATURES=sample_only
...
error: cannot produce proc-macro for `async-trait v0.1.77` as the target `x86_64-unknown-linux-gnu` does not support these crate types
make: *** [Makefile:35: cli] Error 101
```
An issue is raised for this, it seems: https://github.com/rust-lang/rust/issues/71651
In the meantime, this suggests that an extra Makefile target might be needed, though.

Reply:
hmm.... I think it is okay to leave your first implementation